renovate.json

```diff
@@ -1,18 +1,11 @@
 {
-  "extends": [
-    "config:base"
-  ],
-  "rangeStrategy": "replace",
-  "ignorePaths": [],
-  "includePaths": [
-    "package.json",
-    "packages/**",
-    "examples/**"
-  ],
+  "extends": ["config:base"],
+  "includePaths": ["package.json", "packages/**", "examples/**"],
+  "label": "renovate",
   "packageRules": [
     {
-      "updateTypes": ["minor", "patch"],
-      "automerge": true
+      "automerge": true,
+      "automergeType": "branch"
```

Comment:
this change prevents PRs from being created in case CI passes. If CI does NOT pass, a PR is created so devs can look into why CI failed.